packages/wdio-devtools-service/src/gatherer/coverage.ts

```diff
@@ -1,6 +1,5 @@
 import fs from 'fs'
 import path from 'path'
-import atob from 'atob'
```

Comment:
Actually I search across codebase and didn't found any usage of atob, but maybe I missing something. That's why I didnt remove it from package.json

Reply:
we can remove it then from the package.json

Reply:
I removed atob everywhere except one place: [/webiste/package-lock.json](https://github.com/webdriverio/webdriverio/blob/main/website/package-lock.json). I don't why there is atob presented in lock file but missing in package.json. I didn't work with monorepositories and lerna, my be I don't why it works like that